From ad097666e0179d6bea7512b009b0a2fb06c4b9fe Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Thu, 5 May 2016 17:58:48 +0300
Subject: [PATCH 1122/1240] pinctrl: a8040: set RGMII-0 MPP's for CP1

[0-11]  RGMII0
[32-62] = 0xff: Keep default CP1_shared_pins
align rest of MPP[12-31] with A7040-db-router

Change-Id: Ia0512032f1371788696445f5938918498eea64cd
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29644
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/dts/armada-80x0-db.dtsi | 26 +++++++++-----------------
 1 file changed, 9 insertions(+), 17 deletions(-)

diff --git a/arch/arm/dts/armada-80x0-db.dtsi b/arch/arm/dts/armada-80x0-db.dtsi
index a3983fd..f0c17a3 100644
--- a/arch/arm/dts/armada-80x0-db.dtsi
+++ b/arch/arm/dts/armada-80x0-db.dtsi
@@ -234,27 +234,19 @@
 			};
 			pinctl@440000 {
 					/* MPP Bus:
-					   [0-31] = 0xff: Keep default CP0_shared_pins:
-						[11] CLKOUT_MPP_11 (out)
-						[23] LINK_RD_IN_CP2CP (in)
-						[25] CLKOUT_MPP_25 (out)
-						[29] AVS_FB_IN_CP2CP (in)
-					   [32,34] SMI
-					   [31]    GPIO: push button/Wake
-					   [35-36] XSMI
-					   [37-38] I2C
-					   [40-41] SATA[0/1]_PRESENT_ACTIVEn
-					   [44-55] RGMII1
-					   [56-62] SD
+					  [0-11]  RGMII0
+					  [27,31] GE_MDIO/MDC
+					  [32-62] = 0xff: Keep default CP1_shared_pins:
 					*/
 					/*   0    1    2    3    4    5    6    7    8    9 */
-				pin-func = < 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff
+				pin-func = < 0x3  0x3  0x3  0x3  0x3  0x3  0x3  0x3  0x3  0x3
+					     0x3  0x3  0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff
+					     0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x8  0xff 0xff
+					     0x8  0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff
 					     0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff
 					     0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff
-					     0xff 0    7    0    7    7    7    2    2    0
-					     0    0    0    0    1    1    1    1    1    1
-					     1    1    1    1    1    1    0xE  0xE  0xE  0xE
-					     0xE  0xE  0xE>;
+					     0xff 0xff 0xff>;
+
 			};
 
 			comphy {
-- 
1.9.1

